Documentation.md

```diff
@@ -0,0 +1,52 @@
+# Documentation
+In 1929, paper titled " A relation between Diatance and Radial Velocity among extra-Galactic Nebulae" explored 
+the relationship between the distances of galaxies and their radial velocities [1]. Edwin Hubble classified galaxies
+into nine groups based on their distances and redshifts.
+## Backgrounf information
+### *Cepheid Variable stars as Distance Indicator:
+To determine the distances of galaxies, Hubble used Cepheid variable stars, which vary in brightness in a regular
+and predictable way [2]. Henrietta Swan Leavitt had previously discovered a correlation between a Cepheid's brightness
+and its pulsation period, providing a reliable way to calculate distances. By identifying Cepheid stars in nearby
+galaxies, Hubble was able to calculate their absolute brightness and by comparing it to the apparent brightness, 
+derive an approximate distance to each galaxy.
+### * Redshift and Doppler Effect for measuring Radial Velocity:
+Hubble measured the redshift of light from each galaxy, a phenomenon caused by the Doppler effect, which causes
+light from receding objects to shift toward longer (redder) wavelengths. The radial velocity of galaxies could
+thus be inferred from the degree of redshift. Hubble also utilized data from astronomer Vesto Melvin Slipher,
+who had spent years collecting redshift information for various galaxies.
+## Selection of Nine Groups
+### * Grouping Based on Distance Ranges [3]:
+Hubble created these nine groups by organizing galaxies in ranges of increasing distance from Earth. He did
+this to make a clear comparison between the radial velocities (redshifts) and the distances. Each group
+represented a set of galaxies within a similar distance range, allowing him to observe whether galaxies in
+each group had consistent redshift characteristics.
+### * Grouping Based on Pattern of Increasing Redshift with Distance:
+By plotting the radial velocity against distance for each group, Hubble noticed a clear trend: the farther
+a galaxy was, the higher its redshift meaning it was moving away from Earth at a faster speed
+The Linear Relationship: Hubble’s Law
+Through these nine groupings, Hubble found that there was a roughly linear relationship between distance
+and velocity, which he formulated as:
+**$\displaystyle{v}={H}_{{0}}\cdot{d}$**
+where:
+v is the radial velocity (redshift) of a galaxy,
+d is the distance to the galaxy,
+$\displaystyle{H}_{{0}}$ is the constant of proportionality, now known as the Hubble constant.
```

Comment:
Super minor but ```$v = H_0 \cdot d$``` is way simpler and works just fine. Looks like: $v = H_0 \cdot d$

Reply:
I have corrected this as well.